bindata/network-diagnostics/network-check-target.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: network-check-target
+  namespace: openshift-network-diagnostics
+spec:
+  selector:
+      matchLabels:
+        app: network-check-target
+  template:
+    metadata:
+      labels:
+        app: network-check-target
+    spec:
+      containers:
+      - image: openshift/hello-openshift
+        imagePullPolicy: IfNotPresent
+        name: network-check-target-container
+        ports:
+        - containerPort: 8080
+          protocol: TCP
+        resources: {}
+        terminationMessagePath: /dev/termination-log
```

Comment:
is this correct? do we write a real termination message? or should we pull it from stderr?

Reply:
yeah, will change this. It's some daemonset boilerplate i got from somewhere